ui/src/pages/Dashboard/consts.ts

```diff
@@ -0,0 +1,3 @@
+export const FEATURE_FLAGS = {
+    enableSmartSourceDS: true,
+};
```

Comment:
```suggestion
} satisfies FeatureFlags;
```

it does not makes sense now, but maybe they will add more strict types in future